lib/src/main/java/graphql/nadel/validation/NadelValidationContext.kt

```diff
@@ -20,180 +11,16 @@ data class NadelValidationContext internal constructor(
     val hydrationUnions: Set<String>,
     val namespaceTypeNames: Set<String>,
     val combinedTypeNames: Set<String>,
-    val definitions: Map<NadelSchemaMemberCoordinates, List<NadelDefinition>>,
+    val instructionDefinitions: NadelInstructionDefinitionRegistry,
```

Comment:
Created `NadelInstructionDefinitionRegistry` holder and move all the `isHydrated` etc methods into it